### PR TITLE
PSG Auto Sniper

### PR DESCRIPTION
**Pull request by David-Stormwell**
## About The Pull Request

Adds a brand new will be req only NV IFF automatic sniper rifle! It shares the same stats as the normal bolt action, but is a tad slower with a 1.75 second fire delay vs the bolt action's 1.35.
Removes the old Sniper spec from req and replaces it with this sniper.

## Why It's Good For The Game

It's been planed for a while, and more guns that fit the theme for marines to work towards when getting points and miners up as a treat!

## Changelog
:cl:
add: Added a Automatic NV, IFF Sniper rifle. Auto Sniper sprite courtesy of Kuro.
imageadd: added some icons for the Auto Sniper and NV scope. NV scope courtesy of TheBrimmer231. 
imagedel: Replaced the old NV scope with the new one.
/:cl:

### CODE REVIEW

**Comment by ilikesaying** [comment]
David is going on another genocide spree

**Comment by SilencedMP5A5** [comment]
does it still have lase

**Review by TheBimmer231 — APPROVED** [review]
I will not approve this sprite until a shorter one Is made, I already voiced my concern over the length but I guess It was ignored, too bad.

**Review by TheBimmer231 — CHANGES_REQUESTED** [review]
fuck I actually approved, my bad, but yeah, ping me.

**Comment by David-Stormwell** [comment]
> 
> 
> does it still have lase

No

**Comment by SilencedMP5A5** [comment]
kinda bad then ngl

**Review by TheBimmer231 — APPROVED** [review]
still fucking long as hell, but acceptable.

**Review by TiviPlus — APPROVED** [review]

File: code/modules/projectiles/guns/rifles.dm [hunk 1]
```diff
@@ -908,3 +908,47 @@
 /obj/item/weapon/gun/rifle/chambered/reload_into_chamber(mob/user)
 	. = ..()
 	racked_bolt = FALSE
+
+//-------------------------------------------------------
+//T-81 Auto-Sniper
+
+/obj/item/weapon/gun/rifle/standard_autosniper
+	name = "\improper T-81 Automatic Sniper Rifle"
```
Comment (XSlayer300) [inline code/modules/projectiles/guns/rifles.dm:916]:
```suggestion
	name = "\improper T-81 automatic sniper rifle"
```

File: code/modules/projectiles/guns/rifles.dm [hunk 2]
```diff
@@ -908,3 +908,47 @@
...
+	desc = "The T-81 AS is the TerraGov Marine Corps Automatic Sniper Rifle. It is rather well-known for it's night vision scope and IFF ammo, it however lacks a burst fire mode. It is mostly used by people who prefer to do more careful shooting than most. Uses 8.6x70mm caseless IFF caliber."
```
Comment (XSlayer300) [inline code/modules/projectiles/guns/rifles.dm:917]:
```suggestion
	desc = "The T-81 is the TerraGov Marine Corps's automatic sniper rifle. It is rather well-known for it's night vision scope and IFF ammo, it however lacks a burst fire mode. It is mostly used by people who prefer to do more careful shooting than most. Uses 8.6x70mm caseless IFF caliber."
```

File: code/modules/projectiles/magazines/rifles.dm [hunk 3]
```diff
@@ -293,3 +293,14 @@
 	default_ammo = /datum/ammo/bullet/sniper/pfc
 	max_rounds = 7
 	gun_type = /obj/item/weapon/gun/rifle/chambered
+
+//-------------------------------------------------------
+//Marine magazine automatic sniper, or the T-81.
+/obj/item/ammo_magazine/rifle/autosniper
+	name = "T-81 Automatic sniper rifle magazine"
```
Comment (XSlayer300) [inline code/modules/projectiles/magazines/rifles.dm:300]:
```suggestion
	name = "\improper T-81 automatic sniper rifle magazine"
```

**Pull request merged**